Documentation/admission-controller-usage.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Admission Controller
+weight: 2030
+indent: true
+---
+
+# Admission Controller
+
+An admission controller is a piece of code that intercepts requests to the Kubernetes API server prior to persistence of the object, but after the request is authenticated and authorized
+
+ There are two special controllers: MutatingAdmissionWebhook and ValidatingAdmissionWebhook.
+ 
+ Mutating controllers may modify the objects they admit but validation controllers are only allowed to validate requests.
+
+Currently, user can manipulate Custom Resource specs with any values which may result in Rook not functioning as expected. The present validation method in Kubernetes is the OpenAPI schema validation which can be used for basic validations like checking type of data, providing a range for the values etc but anything more complex (checking resource availability, network status, error handling) would not be possible under this scenario.
```

Comment:
This is a better paragraph for a design document, but I think better to remove it for the end user docs.